client/src/sldsLanguageClient.ts

```diff
@@ -76,10 +77,31 @@ function createServerPromise(context: ExtensionContext, outputChannel: OutputCha
 	return new Promise((resolve, reject) => {
 		var server = net.createServer((socket) => {
 			outputChannel.appendLine("SLDS Started");
+			
+			// Temporary solution for an LWC plugin issue where the end character range is too large for SLDS LSP server.
+			let filteredDuplex = new class extends Transform {
+				_transform(chunk: any, encoding: string, callback: TransformCallback) {
+
+					let buf = Buffer.from(chunk).toString();
+					let matcher = /("character":1.7976931348623157e\+308)/;
+				
+					if (matcher.test(buf)) {
```

Comment:
`string.replace` do something similar to `regex.test`. You probably don't need to have both.

Reply:
Thanks @billyma , I fixed this.